hack/smmr-listers.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/vendor/knative.dev/serving/pkg/reconciler/testing/v1alpha1/factory.go b/vendor/knative.dev/serving/pkg/reconciler/testing/v1alpha1/factory.go
```

Comment:
Should we make this one patch together with the serving listers? I assume that's easier to maintain as we'd only have to fix conflicts once, not twice.